src/content/nft/index.md

```diff
@@ -0,0 +1,175 @@
+
+#### The copy/paste problem
+
+Naysayers often bring up the fact that NFTs "are dumb" usually alongside a picture of them screenshotting an NFT artwork. "Look, now I have that image for free!" they say smugly.
+
+Well, yes. But does googling an image of Picasso's Guernica make you the proud new owner of a multi-million dollar piece of art history?
+
+Ultimately owning the real thing is as valuable as the market makes it. The more a piece of content is screen-grabbed, shared, and generally used the more value it gains.
+
+Owning the verifiably real thing will always have more value than not.
+
+### Boosting gaming potential
+
+NFTs have seen a lot of interest from game developers. NFTs can provide records of ownership for in-game items, fuel in-game economies, and bring a host of benefits to the players.
+
+In a lot of regular games you can purchase items for you to use in your game. But if that item was an NFT you could recoup your money by selling it on when you're done with the game. You might even make a profit if that item becomes more desirable.
+
+For game developers – as issuers of the NFT – they could earn a royalty every time an item is re-sold in the open marketplace. This creates a more mutually-beneficial business model where both players and developers earn from the secondary NFT market.
+
+This also means that if a game is no longer maintained by the developers, the items you've collected remain yours under your control.
+
+Ultimately the items you grind for in-game can outlive the games themselves. Even if a game is no longer maintained, your items will always be under your control. This means in-game items become digital memorabilia and have a value outside of the game.
+
+Decentraland, a virtual reality game, lets you buy NFTs representing virtual parcels of land that you can use as you see fit.
+
+<InfoBanner emoji=":eyes:">
+  <div>Check out Ethereum games, powered by NFTs...</div>
+  <ButtonLink to="/dapps?category=gaming">Explore NFT games</ButtonLink>
+</InfoBanner>
+
+### Physical items
+
+...
+
+### Making ethereum addresses more memorable
+
+One of the most useful NFTs developed so far is the Ethereum Name Service. This uses an NFT to provide your Ethereum address with an easier-to-remember name like `mywallet.eth`. This means you could ask someone to send you ETH via `mywallet.eth` rather than `0x123456789.....`.
+
+This works in a similar way to a website domain name which makes an IP address more memorable. And like domains, ENS names have value, usually based on length and relevance. With ENS you don't need a domain registry in order to facilitate the tranfer of ownership. Instead, ENS names can be traded on NFT marketplaces with ease.
+
+ENS names can also point to websites – not just wallets. For example, if you visit ethereum.eth in a web3 browser, it will open up a version of this website.
+
+## NFTs and Defi
+
+NFT-backed loans.
+Minting insurance cover as an NFT that's tradeable on secondary markets....
```

Comment:
Perhaps something about using NFT's as collateral to obtain loans. If you don't have much cash or crypto, but you own valuables, these can potentially be used as collateral to obtain a loan in defi.